godtools/Views/TractElements/TextContent.swift

```diff
@@ -76,29 +66,35 @@ class TextContent: BaseTractElement {
     
     // MARK: - Helpers
     
-    fileprivate func loadElementAttributes(properties: Dictionary<String, Any>) -> (text: String, backgroundColor: UIColor) {
-        var text: String = ""
-        var backgroundColor: UIColor?
+    func loadStyles() {
+        self.properties = (self.parent?.textStyle())!
         
-        if properties["value"] != nil {
-            text = properties["value"] as! String
+        if self.properties.width > CGFloat(0.0) {
+            self.contentWidth = self.properties.width
+        } else {
+            self.contentWidth = (self.parent?.width)!
         }
         
-        if properties["backgroundColor"] != nil {
-            let backgroundColorText = properties["backgroundColor"] as! String
-            switch backgroundColorText {
-            case "red":
-                backgroundColor = .gtRed
-            case "black":
-                backgroundColor = .gtBlack
-            default:
-                backgroundColor = .clear
+        self.xPosition = self.properties.xMargin
+        self.yPosition = self.yStartPosition + self.properties.yMargin
+    }
+    
+    func loadElementProperties(properties: [String: Any]) {
+        for property in properties.keys {
+            switch property {
+            case "value":
+                self.properties.value = properties[property] as! String?
```

Comment:
I think you could do this here instead:
````swift
self.properties.value = properties[property] as? String
````
which is a little bit cleaner and easier to read. Very minor thing.